_layouts/main.html

```diff
@@ -58,14 +58,21 @@ <h2>Meetings</h2>
 
 
  <div class="col-lg-4">
-  <h2>HOW2019</h2>
+  <h2>EPPSU</h2>
     <p>
-      The HSF workshop at Jefferson Lab was a great success. Read our
-      <a href="/newsletter/2019/04/03/Jlab-Workshop.html">newsletter</a>
-      for an overview of all the topics we discussed.
+      HSF has been contributing to the 
+      <a href="https://europeanstrategyupdate.web.cern.ch">European Strategy Update</a> process,
+      emphasising the importance of software for the field.
+    </p>
+    <img src="/images/others/eppsu.png"
+    alt="EPPSU logo" style="display: block; width:40%; margin-left: auto; margin-right: auto;"></a>  
```

Comment:
As said by the tests, there should be not '</a>' here...

Reply:
Yep - thank you link checker!